house/views.py

```diff
@@ -95,3 +95,17 @@ def house_create(request):
         house_form = HouseCreationForm()
 
     return render(request, HOUSE_CREATE_ROUTE, {'form': house_form, 'msg': error_message})
+
+
+@login_required
+def edit_house(request, id):
+    house = get_object_or_404(House, pk=id)
```

Comment:
please use the user object from the request to get the house id
This way other users cant use this API and it will be more secure

Reply:
fixed